code/modules/admin/verbs/debug.dm

```diff
@@ -949,3 +949,68 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 
 /proc/cmp_timer_data(list/a, list/b)
 	return b["count"] - a["count"]
+
+/client/proc/check_missing_sprites()
+	set category = "Debug"
+	set name = "Debug Worn Item Sprites"
+	set desc = "We're cancelling the Spritemageddon. (This will create a LOT of runtimes! Don't use on a live server!)"
+
+	var/actual_file_name
+	for(var/T in subtypesof(/obj/item))
+		var/obj/item/x = new T
```

Comment:
dont instantiate, just use Initial()

Reply:
Planning on doing this change?
Since this is a debug verb not sure it's entirely necessary, but I'd add this only with the `TESTING` define then.

Reply:
I believe I set this up right but if not ping me and let me know,